shared/xccdf/system/auditing.xml

```diff
@@ -896,7 +896,7 @@ cui="3.3.1" />
 
 <Rule id="file_ownership_var_log_audit" severity="medium" prodtype="rhel7">
 <title>System Audit Logs Must Be Owned By Root</title>
-<description>
+<description>Checks that all /var/log/audit files and directories are owned by the root user and group.
 <fileowner-desc-macro file="/var/log" owner="root"/>
```

Comment:
Build the content to check out what `fileowner-desc-macro` does. In essence it will generate filler text about some file having some owner. `/var/log/audit` is a directory, isn't it? The description may not fit.